src/tools/general.js

```diff
@@ -64,23 +64,35 @@ async function resolveInBatches(promiseArray, batchLength = 10) {
 }
 
 const processReview = (review, remoteId) => {
-    const { text, title, rating, tripInfo, publishedDate, userProfile } = review;
+    const { text, title, rating, tripInfo, publishedDate, userProfile, photos } = review;
     const stayDate = tripInfo ? tripInfo.stayDate : null;
     let userLocation = null;
     let userContributions = null;
+    let userName = null;
+    let imageUrls = null;
 
     log.debug(`Processing review: ${title}`);
     if (userProfile) {
-        const { hometown, contributionCounts = {} } = userProfile;
+        const { hometown, contributionCounts = {}, username } = userProfile;
         const { sumReview } = contributionCounts;
         userContributions = sumReview;
         userLocation = hometown.fallbackString;
+        userName = username;
 
         if (hometown.location) {
             userLocation = hometown.location.additionalNames.long;
         }
     }
 
+    if (photos && photos.length > 0) {
```

Comment:
adding photos should be an option on the INPUT_SCHEMA

Reply:
Should open a new issue for new improvement. It's not scope of this bug.